iocore/net/SSLConfig.cc

```diff
@@ -528,3 +516,59 @@ SSLCertificateConfig::release(SSLCertLookup *lookup)
 {
   configProcessor.release(configid, lookup);
 }
+
+void
+SSLTicketParams::LoadTicket() const
+{
+  ticket_block_free(default_global_keyblock);
```

Comment:
I think you should do this last, in case something goes wrong, at least you have the old keyblock. It's another reason to use `std::unique_ptr` - as with `ats_scoped_str` putting a new block of memory in automatically and simultaneously frees the old block.